HW-11/HW12/Homework_12.py

```diff
@@ -0,0 +1,71 @@
+import json
+from datetime import datetime
+
+# 1
+with open('phonebook.txt', 'r+', encoding='utf-8') as phone1:
+    try:
+        phone_book = json.load(phone1)
+    except Exception:
+        phone_book = []
+
+while True:
+    commands = input('Write command:\n')
+    if commands == 'stats':
+        print(f'Amount of records is {len(phone_book)}')
+    elif commands == 'list':
+        [print(*i.keys(), sep='\n') for i in phone_book]
```

Comment:
Here is how this command looks in your original phonebook `print(*phone_book.keys(), sep='\n')`